Source/AssistantV1/Models/CaptureGroup.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * Copyright IBM Corporation 2019
```

Comment:
From this diff, it appears that there are some copyright changes that went into master that didn't get back-ported to the maint branch.  (Just submitted a PR to fix that.)

Here and elsewhere you'll want to revert the change to the copyright statement to preserve the "start year".

I wrote a tool to have find a fix these.  I'm not sure how well it will work with the old copyright format, but it's worth giving it a try.

https://github.ibm.com/mkistler/lint-copyright-dates

Reply:
@mkistler let's discuss. In this case the updated copyrights did come from the copyright linter tool, so if these are incorrectly formatted we'll want to discuss why that happened.

Reply:
Edit: my bad - the copyrights were not applied to this PR. I will add a commit that fixes the formatting.